configure.ac

```diff
@@ -40,6 +40,14 @@ AM_INIT_AUTOMAKE([-Wall -Wno-portability foreign])
 
 FLAGS="-Wall -Wextra -mzarch"
 
+dnl --- check for perl-FindBin module
```

Comment:
I guess you also need to check for perl being installed first, and also make sure that the $PERL variable is set:
```
AC_MSG_CHECKING([if perl is installed])
AC_PATH_PROG(PERL, perl)
if test -z "$PERL" ; then
   AC_MSG_ERROR([Please install perl])
fi
```

Reply:
Yes, good point ... done.